didcomm_resolver/resolver.py

```diff
@@ -133,6 +140,8 @@ async def _resolve(self, profile: Profile, did: str) -> dict:
                         result = json.loads(result)
                     return result
                 except DIDNotFound:
+                    exception_message.format(conn_id + ", {}")
                     continue
 
-            raise DIDNotFound("DID not found on any resolver connections")
+            exception_message = exception_message.replace(", {}", "")
```

Comment:
I think we're better off just saving to a list and then joining the list at the end with 
```python
", ".join(not_found_conn_ids)
```
or something similar.